omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -28,6 +28,17 @@ def fail_with error
     end
   end
 
+  def first_run?
```

Comment:
What do you think, @marcparadise? The tests still pass if we make this change. @stevendanna, FYI, Marc and I messed with this a lot trying to figure out what was most clear.